paddle/scripts/paddle_build.sh

```diff
@@ -451,10 +451,10 @@ function assert_api_spec_approvals() {
       if [ ${API_CHANGE} ] && [ "${GIT_PR_ID}" != "" ]; then
           # NOTE: per_page=10000 should be ok for all cases, a PR review > 10000 is not human readable.
           APPROVALS=`curl -H "Authorization: token ${GITHUB_API_TOKEN}" https://api.github.com/repos/PaddlePaddle/Paddle/pulls/${GIT_PR_ID}/reviews?per_page=10000 | \
-          python ${PADDLE_ROOT}/tools/check_pr_approval.py 1 2887803`
+          python ${PADDLE_ROOT}/tools/check_pr_approval.py 2 2887803 35982308`
```

Comment:
I think only API.spec change need PM approval. Other files, like op_desc.h doesn't need PM approval.

Reply:
I think you're right. I added an if judgment.